persist/adapter.go

```diff
@@ -37,10 +37,15 @@ func LoadPolicyLine(line string, m model.Model) {
 		return
 	}
 
-	key := tokens[0]
+	LoadPolicy(tokens, m)
+}
+
+// LoadPolicy loads a policy rule to model.
+func LoadPolicy(rule []string, m model.Model) {
```

Comment:
Modify the method name to avoid confusion with the following interface

Reply:
The scope of the two is different, I think it does not conflict.

Reply:
Problems with code readability. Don't be ambiguous